roosevelt.js

```diff
@@ -44,6 +47,19 @@ module.exports = function(params) {
 
   console.log(('💭  ' + 'Starting ' + app.get('appName') + ' in ' + app.get('env') + ' mode...').bold);
 
+  // check if env comes from cmd line then leave as is, else use params
```

Comment:
I'm confused here. The point of the param is to handle the case when you don't have a command line flag to set the environment